src/components/qa/exercise-data-filter.cjsx

```diff
@@ -0,0 +1,29 @@
+_ = require 'underscore'
+
+{EcosystemsStore} = require '../../flux/ecosystems'
+
+BOOK_MAPPINGS =
+  '02040312-72c8-441e-a685-20e9333f3e1d': 'stax-soc'
```

Comment:
Hm, I'm not aware of any place that the book UUID's are mapped to tag id's (`stat-bio`, etc.)

Are they?